### PR TITLE
Tweak colors

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206324069027286

### CODE REVIEW

**Comment by paullinator** [comment]
Screenshot 2024-01-12 at 00 45 16
Screenshot 2024-01-12 at 00 43 29
Screenshot 2024-01-12 at 00 42 52
Screenshot 2024-01-12 at 00 42 31

**Comment by paullinator** [comment]
Screenshot 2024-01-13 at 14 03 20
Screenshot 2024-01-13 at 14 04 46
Screenshot 2024-01-13 at 14 05 02

File: src/types/Theme.ts
Comment (Jon-edge) [inline src/types/Theme.ts:1]: Remove Ui4 from keys
https://app.asana.com/0/1200382638405084/1206315466361658/f

File: src/components/ui4/ModalUi4.tsx
Comment (Jon-edge) [inline src/components/ui4/ModalUi4.tsx:1]: I noticed discrepancies between Android and iOS screens. Can we add cases to make it consistent? I noticed that only thing here specific to Android is the `BlurView`'s `overlayColor`, and we can probably make it look more consistent if we case out `modalBackgroundUi4`

**Review by Jon-edge — CHANGES_REQUESTED** [review]

**Review by Jon-edge — APPROVED** [review]

**Pull request merged**